TextConverters/RTF/RTFConsumer.m

```diff
@@ -595,6 +595,14 @@ - (NSAttributedString*) parseRTF: (NSData *)rtfData
       // FIXME: Code page 850 kCFStringEncodingDOSLatin1
       encoding = NSISOLatin1StringEncoding;
     }
+  else if (strncmp(buffer, "ansi", 4) == 0)
```

Comment:
Please have a look at the change to RTF parsing that I just committed on GNUstep master.